local/compose/convergence.go

```diff
@@ -352,13 +355,40 @@ func (s *composeService) connectContainerToNetwork(ctx context.Context, id strin
 		Aliases:           aliases,
 		IPAddress:         ipv4ddress,
 		GlobalIPv6Address: ipv6Address,
+		Links:             links,
 	})
 	if err != nil {
 		return err
 	}
 	return nil
 }
 
+func (s *composeService) getLinks(ctx context.Context, service types.ServiceConfig, projectName string, number int) ([]string, error) {
+	cState, err := GetContextContainerState(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	links := []string{}
+	for _, serviceLink := range service.Links {
+		s := strings.Split(serviceLink, ":")
+		// fixme: alias is optional
+		containers := cState.GetContainers()
+		depServiceContainers := containers.filter(isService(s[0]))
+		fmt.Fprintf(os.Stdout, "------------ service containers ARE HEREE: %+v\n", depServiceContainers)
+		for _, container := range depServiceContainers {
+			fmt.Fprintf(os.Stdout, "------------  containers IS HEREE: %+v\n", container)
+
+			newLink := fmt.Sprintf("%s:%s", fmt.Sprintf("%s_%s_%d", projectName, s[0], number), s[1])
```

Comment:
Tiis part won't because it's not the correct service. 😅 
```suggestion
			newLink := fmt.Sprintf("%s:%s", container.ID, s[1])
```